packages/app/pages/course/[cid]/today.tsx

```diff
@@ -68,11 +60,7 @@ export default function Today() {
             <Col md={12} xs={24}>
               <Row justify="space-between">
                 <Title>Current Office Hours</Title>
-                {role === Role.TA && (
-                  <CreateQueueButton type="default" size="large">
-                    Create Queue
-                  </CreateQueueButton>
-                )}
+                {role === Role.TA && <TACheckinButton courseId={Number(cid)} />}
               </Row>
               {data?.queues?.map((q) => (
                 <OpenQueueCard
```

Comment:
jk i did have one thing, it seems when i join a queue as a student, the # in queue counter on the today page doesn't update now, i believe this was working before tho